### DISCUSSION TITLE
How to make cameraMove

### DISCUSSION THREAD

**Discussion by Mlaofmd** [YoshiCrafter29/YoshiCrafterEngine#156] [Q&A] [answered]
hello i need make a cameraMove on Yoshi engine how look like sonic.exe 2.0

**Comment by BernardoGP4504** [comment, marked as answer]
go into the chart editor and right at the bottom you should have a little box like that:
![box](https://user-images.githubusercontent.com/101405920/168186699-ac44f85e-5714-4ee2-84eb-724ce5610cba.png)
at every section of the chart you want the camera to change change the slider
![ezgif com-gif-maker](https://user-images.githubusercontent.com/101405920/168188562-d3712774-b843-4857-bc08-f043d8086e35.gif)
captions explained everything to say
and also check `[ ] Duet Camera` for every camera change
(this changes the camera in only the section you're in (ex: if you are in section #4 then the camera applies only to that section)

**Reply by BernardoGP4504** [reply]
also you can check `[ ] Must Hit Section` box to make the camera focus on bf (also uncheck `[ ] Duet Camera`)

**Reply by Mlaofmd** [reply]
THANKS!
